server/server/src/main/kotlin/org/jetbrains/bsp/bazel/server/BazelBspServer.kt

```diff
@@ -1,5 +1,6 @@
 package org.jetbrains.bsp.bazel.server
```

Comment:
If this is a change that should be upstreamed, we should merge it into `main` instead of `lucidsoftware-rules-scala-support`.

Reply:
I think it _should_ be upstreamed since it re-implements the bsp spec, but it does seem like they maybe wanted this how it is because intellij does something different. Should I try to make this pr to the other repo anyway do you think?

Reply:
I'm gonna go ahead and merge this into our fork for now. I think it will be easier to upstream this all once we know what we have is working, especially since we are currently trying to get our rules_scala upstreamed.